pkgs/development/libraries/gdk-pixbuf/default.nix

```diff
@@ -44,16 +48,18 @@ stdenv.mkDerivation rec {
     pkg-config
     gettext
     python3
-    gobject-introspection
     makeWrapper
     glib
-    gi-docgen
 
     # for man pages
     libxslt
     docbook-xsl-nons
     docbook_xml_dtd_43
-  ] ++ lib.optional stdenv.isDarwin fixDarwinDylibNames;
+  ] ++ lib.optional stdenv.isDarwin fixDarwinDylibNames
+    ++ lib.optionals withIntrospection [
+    gi-docgen
```

Comment:
Huh, is this still not cross-compatible? It should not rely on introspection.

Reply:
I'll take a look at this.

Reply:
It doesn't fail the build, but I'm getting this warning:

```
Found CMake: NO
Run-time dependency gi-docgen found: NO (tried pkgconfig and cmake)
Looking for a fallback subproject for the dependency gi-docgen

|Executing subproject gi-docgen method meson 
|
|Project name: gi-docgen
|Project version: 2021.3
|Program python3 (jinja2, markdown, markupsafe, pygments, toml, typogrify) found: NO
|
|subprojects/gi-docgen/meson.build:10:0: Exception: python3 is missing modules: jinja2, markdown, markupsafe, pygments, toml, typogrify
```

So I don't think much happens with it either way.

I will re-add it to the buildInputs though.

Reply:
That sounds like pkg-config cannot find gi-docgen.

Reply:
Yes indeed, not sure how to solve that though. Moving it to depsBuildBuild (no change) or buildInputs (gi-docgen doesn't cross-build) doesn't seem to help.

Any ideas?